File: examples/missing/example_missing.R
library("mmstat4")
allbus2018 <- ghload("allbus2018.rds")
vars  <- startsWith(names(allbus2018), "pt")
trust <- allbus2018[,vars]
# count number of missing values
r <- is.na(trust)
# no. per column
apply(r, 2, sum)
# no. per row
apply(r, 1, sum)
# number of complete cases
cc <- complete.cases(trust)
sum(cc)
# filter functions
head(na.omit(trust))
try(na.fail(trust))

File: examples/missing/example_missing.py
import pandas as pd
allbus2018 = ghload("allbus2018.rds")
vars_starting_with_pt = allbus2018.filter(like="pt")
print(vars_starting_with_pt.isnull().sum())         # NA per col
print(vars_starting_with_pt.isnull().sum(axis=1))   # NA per row
print(vars_starting_with_pt.dropna().shape[0])      # complete cases
non_missing_data = vars_starting_with_pt.dropna()
print(non_missing_data.head())
